tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -586,33 +586,37 @@ def _create_dummy_array_for_dot(xp, shape, dtype):
 ]
 
 
-@chainerx.testing.numpy_chainerx_array_equal()
-@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
-@pytest.mark.parametrize('input', [
-    numpy.asarray(0), numpy.asarray(-4), numpy.asarray(4),
-    numpy.asarray(-float('inf')), numpy.asarray(float('inf')),
-    numpy.asarray(float('nan')),
-    numpy.full((), 2), numpy.full((0,), 2), numpy.full((2, 3), 2)
```

Comment:
Why not leave the previous test patterns?
I think it's far better than random values.